cmd/admin-user-add.go

```diff
@@ -75,8 +75,8 @@ EXAMPLES:
 func checkAdminUserAddSyntax(ctx *cli.Context) {
 	argsNr := len(ctx.Args())
 	if argsNr > 3 || argsNr < 1 {
-		fatalIf(errInvalidArgument().Trace(ctx.Args().Tail()...),
-			"Incorrect number of arguments for user add command.")
+		// fix #4326 , display help menu.
+		showCommandHelpAndExit(ctx, "add", 1)
```

Comment:
```suggestion
		showCommandHelpAndExit(ctx, ctx.Command.Name, 1)
```

Reply:
I found the same problem in other help information, and I will fix it together.